applications/management/commands/get_verified_addresses.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand, CommandError
+from applications import models
+import csv
+import easypost
+from time import sleep
+from django.conf import settings
+from app import settings as app_settings
+easypost.api_key = app_settings.EASYPOST_KEY
+
+
+class Command(BaseCommand):
+    help = 'Prints verified addresses for users'
+
+    def handle(self, *args, **options):
+        print('Gathering verified addresses...')
```

Comment:
command output in django needs to be written to stdout instead of just print.
https://docs.djangoproject.com/en/3.1/howto/custom-management-commands/